exts/cesium.omniverse/cesium/omniverse/ui/statistics_widget.py

```diff
@@ -0,0 +1,76 @@
+import carb.events
+import omni.kit.app as app
+import omni.ui as ui
+from typing import List, Optional
+from ..bindings import ICesiumOmniverseInterface
+
+
+class CesiumOmniverseStatisticsWidget(ui.Frame):
+    """
+    Widget that displays statistics about the scene.
+    """
+
+    NUMBER_OF_MATERIALS_LOADED_TEXT = "Number of materials loaded: {0}"
+    NUMBER_OF_GEOMETRIES_LOADED_TEXT = "Number of geometries loaded: {0}"
+    NUMBER_OF_GEOMETRIES_VISIBLE_TEXT = "Number of geometries visible: {0}"
```

Comment:
Generally it's better to define static variables like this at the module level (outside of the class) rather than as a class variable. It hides them from any potential callers and ensures they'll only ever be interpreted once.